apps/frontend/src/libs/components/table/styles.module.css

```
@@ -51,6 +51,10 @@
 	overflow-y: auto;
 }
 
+.table .table-body--no-scroll {
```

we don't use bem in project
```suggestion
.table .table-body-no-scroll {
```